### ISSUE TITLE
Crossed brainstem findings

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10550]
**Preferred term label:**
Crossed brainstem findings

**Synonyms**
Motor and/or sensory cranial nerve findings ipsilateral to the lesion, and at the same time motor and/or sensory findings on the contralateral side of the body

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Wallenberg Syndrome
Medial Medullary Syndrome

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001101
- label: crossed brainstem sign
- parents: [Abnormal central motor function (HP:0011442)](https://hpo.jax.org/app/browse/term/HP:0011442)
- pmid: PMID:19252797|PMID:32809731
- def: Motor and/or sensory cranial nerve findings ipsilateral to the lesion, and at the same time motor and/or sensory findings on the contralateral side of the body.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10550
- synonyms:
